cmd/clusterctl/api/v1alpha3/annotations.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+const (
+	// CertManagerVersionAnnotation reports the cert manager version installed by clusterctl.
+	CertManagerVersionAnnotation = "certmanager.clusterctl.cluster.x-k8s.io/version"
```

Comment:
```suggestion
	CertManagerVersionAnnotation = "cert-manager.clusterctl.cluster.x-k8s.io/version"
```

Reply:
@vincepri this annotation already exists; if we are going to change it we should make the code able to understand both the old one and the new one for a certain period, otherwise we won't recognise the value during upgrades. Is that ok for you?

Reply:
Eh, let's leave it :)

Reply:
Added a commit to with the annotation rename + the code for managing migration